chromium_autostart.desktop

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=Connect
+Comment=Starts Chromium Kiosk
+Exec=/usr/bin/chromium-browser --kiosk http://localhost/
```

Comment:
Would this work?
```bash
Exec=while ! curl localhost; do sleep 1;  /usr/bin/chromium-browser --kiosk http://localhost/
```

Reply:
That is a good idea but it will have to be in a separate shell script.